.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+---
+version: 2
+ updates:
+   - package-ecosystem: "docker"
+     directory: "/"
+      reviewers:
+         - "k3s-io/k3s-dev"
```

Comment:
```suggestion
     reviewers:
        - "k3s-io/k3s-dev"
```

Reply:
Sorry my suggestion was off by a space :upside_down_face:

Reply:
Okay, I've fixed indentation and cleaned commit history.